core/extensions.rs

```diff
@@ -368,6 +388,11 @@ macro_rules! extension {
       }
 
       #[allow(dead_code)]
+      /// Initialize this extension for use with ES modules
+      /// For CommonJS, use init_js_only instead
```

Comment:
```suggestion
      /// Initialize this extension for runtime or snapshot creation. Use this
      /// function if the runtime or snapshot is not created from a (separate)
      /// snapshot, or that snapshot does not contain this extension. Otherwise
      /// use `init_ops()` instead.
```